index.html

```diff
@@ -21,158 +21,174 @@
 
   <body>
     <div class="header">
```

Comment:
This div isn't semantic, it's a generic html tag. Mayble look at the class string as a clue to make it semantic!

Reply:
okay, thank